proof/access-control/ADT_AC.thy

```diff
@@ -95,7 +95,8 @@ lemma do_user_op_respects:
            apply (rule dmo_device_update_respects_Write)
           apply (wpsimp wp: dmo_um_upd_machine_state
                             dmo_user_memory_update_respects_Write
-                            hoare_vcg_all_lift hoare_vcg_imp_lift)+
+                            hoare_vcg_all_lift hoare_vcg_imp_lift
+                    wp_del: select_wp)+
           apply (rule hoare_pre_cont)
          apply (wpsimp wp: select_wp)+
```

Comment:
There's a few places like this where existing proofs made use of the fact that `select_wp` wasn't in the `wp` set and were broken by this change. I could probably restructure the proof to avoid this but for now I've just deleted `select_wp` wherever required for the original proofs to continue working.

Reply:
It looks like you were right and this situation is the rare one, so this is fine by me.

Reply:
I agree